Leanplum-SDK/Classes/Features/Actions/LPActionManager.m

```diff
@@ -765,7 +762,6 @@ - (void)listenForLocalNotifications
         NSString *message = [context stringNamed:@"Message"];
 
         // Don't send notification if the user doesn't have the permission enabled.
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 80000
```

Comment:
Is this removed, assuming that all devices will be 8 and above?

Reply:
yes, because our target is now ios 8, so this code will not get built for <8

Reply:
Sounds good. Can you elaborate on test cases, do we need test cases for this, or will existing test cases suffice . I know this is just upgrading version , so we are in safe zone.

Reply:
current tests all run on the latest version and should be fine